### PR TITLE
feat(semver): drop standard-version + all conventional-changelog config

### PR DESCRIPTION
**Pull request by nowseemee**
Yo, this tool is awesome, thanks for it! 💫💫💫

# Description
I have a project at Azure DevOps Repos (don't ask why) and the default diff link pattern of the generated change log was off. So...

# About the PR:
- I enabled all conventional-changelog config options and while I was at it, 
  - https://github.com/jscutlery/semver/issues/613 
  - https://github.com/jscutlery/semver/issues/457
- I removed the deprecated dependency. 
  - https://github.com/jscutlery/semver/issues/509

### CODE REVIEW

File: packages/semver/src/executors/version/utils/write-changelog.ts [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+    const context = { version: newVersion };
+    const changelogStream = conventionalChangelog(
+      {
+        preset: config.preset || 'angular',
+        tagPrefix: config.tagPrefix,
+      },
+      context,
+      { merges: null, path: config.path } as conventionalChangelog.Options
+    ).on('error', function (err: unknown) {
+      return reject(err);
```
Comment (nowseemee) [inline packages/semver/src/executors/version/utils/write-changelog.ts:67]: I'll add tests here soon...

File: packages/semver/src/executors/version/utils/write-changelog.ts [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+  try {
+    accessSync(config.infile, constants.F_OK);
+  } catch (err: any) {
+    if (err.code === 'ENOENT') {
+      writeFileSync(config.infile, '\n', 'utf8');
+    }
+  }
+
+  const existingContent = buildExistingContent(config);
+  const newContent = await buildConventionalChangelog(config, newVersion);
```
Comment (nowseemee) [inline packages/semver/src/executors/version/utils/write-changelog.ts:24]: will add a catch

File: packages/semver/src/executors/version/index.e2e.spec.ts [hunk 3]
```diff
@@ -1318,6 +1318,39 @@ $`)
...
+          workspaceRoot: testingWorkspace.root,
+        })
+      );
+
+      expect(readFileSync('packages/b/CHANGELOG.md', 'utf-8')).toMatch(
+        new RegExp(`^
+`)
+      );
+      expect(console.info).toHaveBeenCalledWith(
+        expect.stringContaining(`fix emptiness`)
```
Comment (nowseemee) [inline packages/semver/src/executors/version/index.e2e.spec.ts:1349]: this is quite sad, please HALP to add a proper regex for stringMatching

Comment (nowseemee) [reply packages/semver/src/executors/version/index.e2e.spec.ts]: ended up with a simpler unit test

**Comment by edbzn** [comment]
Hey, I'm in hollidays and back pretty soon! I'm excited to check your work. Thanks in advance.

**Comment by nowseemee** [comment]
Hey, I hope you are having a good one! Take your time, work can wait :)

**Review by edbzn — APPROVED** [review]

**Comment by edbzn** [comment]
@nowseemee Thanks a lot for the work!

**Pull request merged**